src/ui/dialog-vkbasalt.ui

```diff
@@ -0,0 +1,819 @@
+        <child>
+          <object class="GtkLabel">
+            <property name="xalign">0</property>
+            <property name="wrap">True</property>
+            <property name="wrap-mode">word-char</property>
+            <property name="ellipsize">none</property>
+            <property name="lines">4</property>
+            <property name="use-markup">True</property>
+            <property name="max-width-chars">40</property>
+            <property name="label" translatable="yes">DLS sharpness increases the sharpness of a frame. Higher values make the frame sharper.</property>
+          </object>
+        </child>
+      </object>
+    </child>
+  </object>
+  <object class="GtkPopover" id="popover_dls_denoise">
+    <child>
+      <object class="GtkBox">
+        <property name="orientation">vertical</property>
+        <property name="spacing">6</property>
+        <property name="margin-start">6</property>
+        <property name="margin-end">6</property>
+        <property name="margin-top">6</property>
+        <property name="margin-bottom">6</property>
+        <child>
+          <object class="GtkLabel">
+            <property name="xalign">0</property>
+            <property name="wrap">True</property>
+            <property name="wrap-mode">word-char</property>
+            <property name="ellipsize">none</property>
+            <property name="lines">4</property>
+            <property name="use-markup">True</property>
+            <property name="max-width-chars">40</property>
+            <property name="label" translatable="yes">DLS denoise decreases the noise of a frame. Higher values make the frame softer.</property>
+          </object>
+        </child>
+      </object>
+    </child>
+  </object>
+  <object class="GtkPopover" id="popover_fxaa_subpixel_quality">
+    <child>
+      <object class="GtkBox">
+        <property name="orientation">vertical</property>
+        <property name="spacing">6</property>
+        <property name="margin-start">6</property>
+        <property name="margin-end">6</property>
+        <property name="margin-top">6</property>
+        <property name="margin-bottom">6</property>
+        <child>
+          <object class="GtkLabel">
+            <property name="xalign">0</property>
+            <property name="wrap">True</property>
+            <property name="wrap-mode">word-char</property>
+            <property name="ellipsize">none</property>
+            <property name="lines">4</property>
+            <property name="use-markup">True</property>
+            <property name="max-width-chars">40</property>
+            <property name="label" translatable="yes">FXAA subpixel quality decreases aliasing at the subpixel level. Higher values make the frame softer.</property>
+          </object>
+        </child>
+      </object>
+    </child>
+  </object>
+  <object class="GtkPopover" id="popover_fxaa_quality_edge_threshold">
+    <child>
+      <object class="GtkBox">
```

Comment:
I think one does not need a box here, just put the label directly as a child of the popover with the properties of the box on the label.

On the other hand `orientation=vertical` is wrong here and might cause issues. (Never set orientation on a box containing a single widget)

If you really need a container for a single widget you can use `AdwBin`.

Reply:
> I think one does not need a box here, just put the label directly as a child of the popover with the properties of the box on the label.

If we remove the box, it won't have any padding:
![image](https://user-images.githubusercontent.com/50847364/179878991-7f0721f6-13a8-464d-89e2-86c142abea31.png)

Reply:
...I'm not sure why I haven't thought of that.